src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -46,7 +46,9 @@ export const WithControls: Story = {
   },
   render: (args) => (
     <Tooltip {...args}>
-      <Button id="tooltip-button">Hover me</Button>
+      <Button height="40px" id="tooltip-button">
```

Comment:
This makes me think that the height should be added to the `Button` component instead of here multiple times.